docs/core-concepts/working-with-templates/data-objects.md

```diff
@@ -0,0 +1,47 @@
+# Data Available in Templates
+
+Apostrophe makes different data available for you in templates depending on the context. Most of this is covered in other sections as well, but this page serves to collect that information in one place, as well as highlight options not covered elsewhere.
+
+::: tip
+Anything attached to the `req.data` object becomes visible as the `data` object in Nunjucks templates when rendering pages and other templates. Before a page is rendered, you can add properties onto `req.data` to make any new information that you need available in a template.
+:::
+
+## `data.global`
+
+`data.global` is available everywhere and always represents the same `apostrophe-global` document with the same information everywhere. Read more about this on the [global doc reference page](/core-concepts/global-settings/global.html#the-global-doc-sharing-content-across-pages).
+
+## `data.page`
+
+The `data.page` object is available in any page, piece "show page" (e.g., `show.html` -- these aren't "pages" in the sense that they might extend `apostrophe-pages`), and widget template. It contains information about the current page's document (e.g., title, areas). There is a bit more to think about if  you are using it on show page or widget templates, but most of the time you will probably be using this directly on a normal page or index page template.
```

Comment:
Taking a crack at replacing the first sentence here:

"The `data.page` object is available in any template rendered in response to loading a page in Apostrophe. This is true even if there is additional information in the URL, such as the slug of a piece (like a blog post) beyond the slug of the page. In that case the piece is also available as `data.piece`. In widget templates it is not guaranteed to be available, in particular when a widget is first inserted in a page or dialog box the entire page is not rendered, so your widget templates should rely only on information passed to them as options by the template or contained in the widget.

Reply:
Also: "When using data.page in a layout template, take care to remember that it may not be present. For instance, in the notFound.html page template it is not present, and that often extends a shared layout."

Reply:
I'll correct what I have here. I don't want to get too obtuse here since the show page/piece stuff is already pretty confusing.